config.xml

```diff
@@ -18,6 +18,7 @@
     <preference name="SplashShowOnlyFirstTime" value="false" />
     <preference name="SplashScreen" value="screen" />
     <preference name="SplashScreenDelay" value="3000" />
+    <preference name="AllowBackForwardNavigationGestures" value="false" />
```

Comment:
Similar to the other question, why was this changed?

Reply:
I dont know, it seems this change was introduced by @shreedhanu08 when creating the new `CheckPairingCode`-screen.

Reply:
Same response as above - as the PR owner, you're responsible for all the changes that you're trying to get merged here.

Reply:
This commit: e7b7a6c330f597ef8dc2c2412abf8f2f6d62ba71

I used a branch where you had created a `PairingCodeScreen`

Reply:
Hey Mads, I think there's a lot of miscommunication / confusion here. I never personally added the `PairingCodeScreen`, and I'm not sure why that was added to the `proxy-updates` branch.

The timeline for the `proxy-updates` branch is particularly curious:

1. 2/23: I created the `proxy-updates` branch from latest `main` (confirmed by my Slack conversation with Sune)
2. 3/3: Sune merged 2 commits into the `proxy-updates` branch through PR #42 
3. 3/7: @shreedhanu08 pushes commit https://github.com/trustthevote/MVP-App-TestHarness/commit/e7b7a6c330f597ef8dc2c2412abf8f2f6d62ba71 (???)
4. 3/8: @shreedhanu08 pushes commit https://github.com/trustthevote/MVP-App-TestHarness/commit/adb1d9d5de648812849c5a60e0295e134497cee7 (???)
5. 3/17: PR #46 is merged, which contains 5 commits from you and Sune

The two commits pushed by @shreedhanu08 appear to have A) not come from any PR (as would be the preferred practice) and B) were not also added on the `main` branch. 

**In short, these commits just "showed up" in the `proxy-updates` branch, and we're all now seeing and reacting to them for the first time. Someone needs to tell us whether they need to be in this branch or not.** If we assume that these commits were not added by accident, that means there was a willful intent to add material changes to the `proxy-updates` branch, and that @shreedhanu08 was likely asked by someone to do so.

Mads, I missed that in my initial analysis, so thanks for bringing this up. This is definitely not a question for you.

@awoflaherty @trustthevote, can you help us with this mystery?

Reply:
Resolution summarized here: https://github.com/trustthevote/MVP-App-TestHarness/pull/48#issuecomment-1093634620